src/main/java/uk/gov/register/util/ElementType.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.register.util;
+
+public enum ElementType {
```

Comment:
I can see why you have this, but I wonder if we should stick with our current pattern of having one page for multiple records/entries/items and another page for a single record/entry/item. This is probably worth some discussion.

Reply:
Discussed.